test/test_client.py

```diff
@@ -148,7 +148,7 @@ async def test_connection_failure(self) -> None:
             await connection.open_group_socket()
         finally:
             await connection.stop()
-            with suppress(asyncio.CancelledError, ConnectionAbortedError):
```

Comment:
If a `ConnectionAbortedError` is raised here (in an explicit `stop()`, I would consider this a bug, so we should not ignore the exception.